packages/execute-driver/lib/execute-child.js

```diff
@@ -0,0 +1,134 @@
+import _ from 'lodash';
+import B from 'bluebird';
+import vm from 'vm';
+import { logger } from '@appium/support';
+import { attach } from 'webdriverio';
+
+const log = logger.getLogger('ExecuteDriver Child');
+
+// duplicate defining these keys here so we don't need to re-load a huge appium
+// dependency tree into memory just to run a wdio script
+export const W3C_ELEMENT_KEY = 'element-6066-11e4-a52e-4f735466cecf';
```

Comment:
I'd say it is already there

Reply:
if it's in `@appium/support`, OK, since we're already importing it. the only place i knew it to be exported was from basedriver, which i thought might have had a large memory footprint and didn't want to bring in just for this little ephemeral operation.

also, it's worth pointing out that all this code was written before, i'm just porting it to a plugin. not to say i'm not open to making it better, but the main point of this PR is just to port it, not to introduce new code.